keyboards/amjpad/amjpad.h

```diff
@@ -23,37 +23,37 @@
  */
 // The first section contains all of the arguments
 // The second converts the arguments into a two-dimensional array
-#define KEYMAP( \
-    k00, k01, k02, k03, \
-	k10, k11, k12, k13, \
-	k20, k21, k22, k23, \
-	k30, k31, k32, \
-	k40, k41, k42, k43, \
-	k50, k52 \
+#define LAYOUT( \
```

Comment:
There are a couple of 6 row keypads.  Maybe we should have some layouts for them?

Reply:
I had actually considered doing just that, but I wasn't sure 1) if it was wanted and a good idea, and 2) what changes it would require of my workflow, if any.

I was thinking of something like the following:

- LAYOUT_ortho_6x4
- LAYOUT_ortho_5x4
- LAYOUT_numpad_6x4
- LAYOUT_numpad_5x4

`numpad` layouts would be 2u `Num 0` with `Num +` and `Num Enter` both vertical.

The caveat I have about these is some numpads are wired where the vertical keys are on the `789` and `123` rows, and some are `456` and `0.` rows. So not sure if the numpad ones are a good idea because of that.

Reply:
Below is what I'm talking about:

![numpad_matrices](https://user-images.githubusercontent.com/18669334/40591453-75a1fbda-61c6-11e8-906a-b08e64a0ec5d.png)

Basically, either the style on the left or the one on the right becomes `LAYOUT_numpad_5x4`, and the other is `LAYOUT_numpad_5x4_offset` or something like that. Keymaps written for one don't work on the other (different number of keys per row), and how do we make this plain/obvious for other people contributing?

This also disregards the possibility someone makes a PCB/matrix where the 2u `Num 0` key is col1 instead of col0. Or some hodgepodge of all the above.

Reply:
The key config is the same, right?  Eg, the placement for users is the same.

In that case, it doesn't matter for the LAYOUT. That's the point.  The reason we use the LAYOUT macro is so that we can remap the matrix. 

So, both of these work on the same layout:
```
#define LAYOUT_numpad_5x4( \
  k00, k01, k02, k03, \
  k10, k11, k12, k13, \
  k20, k21, k22, \
  k30, k31, k32, k33, \
  k40, k42 \
) \
{ \
  {k00, k01, k02, k03}, \
  {k10, k11, k12, k13}, \
  {k20, k21, k22, XXX}, \
  {k30, k31, k32, k33}, \
  {k40, XXX, k42, XXX}, \
}
``` 

```
#define LAYOUT_numpad_5x4( \
  k00, k01, k02, k03, \
  k10, k11, k12, k23, \
  k20, k21, k22, \
  k30, k31, k32, k43, \
  k40, k42 \
) \
{ \
  {k00, k01, k02, k03}, \
  {k10, k11, k12, XXX}, \
  {k20, k21, k22, k23}, \
  {k30, k31, k32, XXX}, \
  {k40, XXX, k42, k43}, \
}
```

The matrix itself is different, but the LAYOUT/keymap is the same for both boards.

Reply:
Though, this would look better: 

```
#define LAYOUT_numpad_5x4( \
  k00, k01, k02, k03, \
  k10, k11, k12, \
  k20, k21, k22, k13, \
  k30, k31, k32,\
  k40, k42,       k33, \
) \
```